docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   simh:
-    build: .
```

Comment:
Can you leave it commented? when changing the dockerfile, we'd like to use this line for building automatically.